native/cocos/core/builtin/DebugInfos.cpp

```diff
@@ -451,6 +451,7 @@ ccstd::unordered_map<int, ccstd::string> debugInfos = {
 { 16001, "'%s' is deprecated since v%s, please use '%s' instead." },
 { 16002, "'%s' is removed since v%s." },
 { 16003, "'%s' is removed since v%s, please use '%s' instead." },
+{ 16101, "The effect('%s') you are looking for does not exist, please confirm the effect name in the editor. NOTE: Since 3.6, the name of the built-in effect has been changed to its name in the editor, please check it out. More information please refer to https://docs.cocos.com/creator/manual/zh/shader/effect-inspector.html" },
```

Comment:
```suggestion
{ 16101, "The effect('%s') you are looking for does not exist, please confirm the effect name in the editor. NOTE: Since 3.6, the name of the built-in effect has been changed to its name in the editor, please check it out. More information please refer to https://docs.cocos.com/creator/manual/en/shader/effect-inspector.html" },
```